src/common/utils/deploy.utils.ts

```diff
@@ -58,13 +59,22 @@ export default class DeployUtils {
     return flags;
   }
 
-  static getComponentSecrets(individual_secrets: string[], secrets_file?: string): Dictionary<Dictionary<string | number | null>> {
-    const component_secrets = DeployUtils.readSecretsFile(secrets_file);
+  static getComponentSecrets(individual_secrets: string[], secrets_file: string | string[]): Dictionary<Dictionary<string | number | null>> {
+    // Check to see if there are multiple secret files; else, just read the single secret file
+    let component_secrets: any = {};
+    for (const secret_file of secrets_file) {
```

Comment:
`secrets_file` can be both a string or string array. This code would work fine for a string array but not a string. Can we make secrets_files only an array here?